### PR TITLE
Update compilation instructions

### PR DESCRIPTION
**Pull request by christophfroehlich**
so that we maybe don't forget it when we branch for K-turtle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,12 +37,12 @@ Note that `gz_ros2_control` depends on the version of Gazebo that is
...
 Currently, for ROS 2 Jazzy and Rolling, the Gazebo version is Harmonic.
 
-To compile `gz_ros2_control` from source, create a workspace, clone the repo and compile it:
+To compile `gz_ros2_control` from source, create a workspace, clone the correct branch of this repo and compile it:
 
 ```bash
 mkdir -p ~/gz_ros2_control_ws/src
 cd ~/gz_ros2_control_ws/src
-git clone https://github.com/ros-controls/gz_ros2_control
+git clone https://github.com/ros-controls/gz_ros2_control -b master
```
Comment (ahcorde) [inline README.md]: we should use `rolling` here. @christophfroehlich do you know if I will break something on your CI is I change this to `rolling`? I need to do the appropriate in rosdistro too.

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (christophfroehlich) [reply README.md]: we'd have to fix a few repos files in ros2_control_ci and demos repo, not a big deal.

Comment (ahcorde) [inline README.md]:
```suggestion
git clone https://github.com/ros-controls/gz_ros2_control -b {ROS_DISTRO}
```

**Review by ahcorde — CHANGES_REQUESTED** [review]
I renamed the branch to `rolling`

**Review by ahcorde — APPROVED** [review]

**Pull request merged**